impl/proto/mutation_v1_service/gen.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate protoc -I=. -I=$GOPATH/src/ -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis/ --go_out=,plugins=grpc:. mutation_v1_service.proto
```

Comment:
I'll simply remove this change as I don't need to change the mutation service proto?